notes/transcript10.md

```diff
@@ -0,0 +1,242 @@
+
+# Sticker Market
+8:00 - 9:00
+
+Oskar: Cool, so sticker market?
+
+Julien: Yep, so last week we mostly worked on values documents, like the result document is done, the specification is done. Mostly because Rachel invested a lot of time in this. We are validated the mockup so I think we are done with the mockup themselves. We agree that this is the right directions. Andrei spent some time implementing the mockup, and this week I will try to glue the mockup with contracts, and by the way Ricardo did some work on this contract too. One eventual open question that I would like to raise, for the MVP we probably will not consider ESC721 but it's still open to this discussion so if someone has an opinion around that, that would be appreciated.
+
+# Dapp Store
+9:00 - 10:00
+
+Oskar: Cool. Dapp Store?
+
+Andy: Yeah the Dapp Store we have the research phase running at least to the end of this month. There are a few different moving pieces to it and it is only P1 so we don't have many core contributors taking parts. There's two major parts to it we've decided the first is simply design in order to solve some of the most basic user stories about being able to discover Dapps easily, navigate to the one's that you've recently used and ones which might be relevant to you based on what's trending across the network. Which for version one will probably be calling an API seen as it's one of many options, not the only option and it if falls over it won't break.
+
+10:00 - 11:00
+
+Andy: Taken the decision to make it as a web dapp in mind with the sticker markets and whatever other depths like that come out of it, and then in parallel we'll be doing a bunch more research around what exactly the curation mechanism might be for something like this, Ricardo's offered a bit of help there. Yeah I put forward a proposal about using simple bonded curves but that obviously still needs to be validated by others, and like I said we've got a little bit of lack of resources on this swarm so hopefully we can experiment with more bounties and sort of external contributions while the P0 work that everybody else is doing goes ahead full-steam.
+
+# Core Network Incentives
+11:00 - 12:00
+
+Oskar: Cool, so network incentives?
+
+Status: Yeah we are continuing our research like Andrea did research on one of the ideas of incentive of the current infrastructure meaning whisper and mail servers. For the next week we wanna also take a look like taken into account that's there's also a protocol research happening so we want something more broad and applicable and future proof. So people want to have some research on something different and not like whisper side, just come up with something that's more generic and durable so that's essentially it I hope we'll share the documents this week, the drafts of research.
```

Comment:
Status should be Igor here (and elsewhere)

Reply:
Fixed on 65